File: backend/components/metiers/threed/security/docs/samples/rgpd/helpers/helpers_rgpd.py
# helpers_rgpd.py
# Helpers avancés pour la conformité RGPD, privacy by design, consentement

def anonymize_data(data):
    """Simule l'anonymisation de données personnelles"""
    return {k: '***' for k in data}

def check_consent(user):
    """Vérifie le consentement utilisateur"""
    return user and user.get('consent') is True

File: backend/components/metiers/threed/security/docs/samples/rgpd/helpers/helpers_rgpd.js
// helpers_rgpd.js
// Helpers avancés pour la conformité RGPD, privacy by design, consentement

function anonymizeData(data) {
  // Simule l'anonymisation de données personnelles
  return Object.fromEntries(Object.entries(data).map(([k, v]) => [k, '***']));
}

function checkConsent(user) {
  // Vérifie le consentement utilisateur
  return user && user.consent === true;
}

module.exports = { anonymizeData, checkConsent };
